config/routes.rb

```diff
@@ -234,8 +234,9 @@ def redirect_with_vary(path)
   get '/about',      to: 'about#show'
   get '/about/more', to: redirect('/about')
 
-  get '/privacy-policy', to: 'privacy#show', as: :privacy_policy
-  get '/terms',          to: redirect('/privacy-policy')
+  get '/privacy-policy',   to: 'privacy#show', as: :privacy_policy
```

Comment:
I bet these two lines would be so happy to be wrapped in a `with_options to: 'privacy#show'` block!?

Reply:
Actually, this benefitted from having a separate controller. Changed.